From 8f3aee6cd1a1d054743698ddb32911645634bfef Mon Sep 17 00:00:00 2001
From: Shenwei Wang <shenwei.wang@freescale.com>
Date: Tue, 18 Nov 2014 10:03:53 -0600
Subject: [PATCH 0976/1543] MLK-9869: ARM: imx_v7_defconfig: add BT config

commit bcfc9af2dc6a6090ac79f63b9d08fafde3868152 from
git://git.freescale.com/imx/linux-2.6-imx.git

added BT config to imx_v7_defconfig to enable the BT interface in Linux
kernel by default.

Signed-off-by: Shenwei Wang <shenwei.wang@freescale.com>
---
 arch/arm/configs/imx_v7_defconfig |   27 +++++++++++++++++++++++++++
 1 files changed, 27 insertions(+), 0 deletions(-)

diff --git a/arch/arm/configs/imx_v7_defconfig b/arch/arm/configs/imx_v7_defconfig
index 70ff2e5..9e12b66 100644
--- a/arch/arm/configs/imx_v7_defconfig
+++ b/arch/arm/configs/imx_v7_defconfig
@@ -71,6 +71,33 @@ CONFIG_CFG80211=y
 CONFIG_MAC80211=y
 CONFIG_RFKILL=y
 CONFIG_RFKILL_INPUT=y
+
+CONFIG_BT=y
+CONFIG_BT_RFCOMM=y
+CONFIG_BT_RFCOMM_TTY=y
+CONFIG_BT_BNEP=y
+CONFIG_BT_BNEP_MC_FILTER=y
+CONFIG_BT_BNEP_PROTO_FILTER=y
+CONFIG_BT_HIDP=y
+
+#
+# Bluetooth device drivers
+#
+CONFIG_BT_HCIBTUSB=y
+CONFIG_BT_HCIBTSDIO=y
+CONFIG_BT_HCIUART=y
+CONFIG_BT_HCIUART_H4=y
+CONFIG_BT_HCIUART_BCSP=y
+CONFIG_BT_HCIUART_ATH3K=y
+# CONFIG_BT_HCIUART_LL is not set
+# CONFIG_BT_HCIUART_3WIRE is not set
+CONFIG_BT_HCIBCM203X=y
+# CONFIG_BT_HCIBPA10X is not set
+# CONFIG_BT_HCIBFUSB is not set
+# CONFIG_BT_HCIVHCI is not set
+# CONFIG_BT_MRVL is not set
+CONFIG_BT_ATH3K=y
+
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
 # CONFIG_STANDALONE is not set
-- 
1.7.5.4

